### DISCUSSION TITLE
Point Blank for GG not working

### DISCUSSION THREAD

**Discussion by rabbitmurias** [3kh0/ext-remover#529] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
When clicking the link for GG for Point Blank, it says the file there is down or has permanently moved to a new address.

**Comment by Aka-but-nice** [comment]
Yes, it doesn't work for goguardian, well anymore.

**Reply by rabbitmurias** [reply]
Does anything work for goguardian now? (other than downgrading)
